swagger.yml

```diff
@@ -2299,6 +2297,13 @@ definitions:
         type: string
       text:
         type: string
+  pluginRun:
```

Comment:
The response includes `deploy_id` but not the request body, which explains this, right?

Reply:
Yep! The response should match the full PluginRun model.